photonjs/README.md

```diff
@@ -161,6 +162,44 @@ type Profile = {
   - Terminating/chainable operation
   - Graph selection: Travesal vs operate
   - ...
+  
+## Debugging
+```ts
+
+// Either like this
+const photon = new Photon({
+  logQueries: 'pipe',
+})
+
+// Or like this
+const photon = new Photon({
+  debug: {
```

Comment:
Not sure whether `debug` is the right term. To me `log` or `logging` sounds more appropriate?

Reply:
Totally. New suggestion: Calling it `log`.